tasks/main.yml

```diff
@@ -1,3 +1,6 @@
 ---
```

Comment:
Why does this file have so many empty lines?

Reply:
fixed